src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -96,6 +96,8 @@
   <ItemGroup>
     <Compile Include="ReadOnlySpan\AsSpan.cs" />
     <Compile Include="ReadOnlySpan\BinarySearch.cs" />
+    <None Include="ReadOnlySpan\Contains.byte.cs" />
+    <None Include="ReadOnlySpan\Contains.T.cs" />    
```

Comment:
Not sure where to put these tests such that they don't get broken by cross-compilation (eg NetFx)

Reply:
I would put them under here (this is where tests for .NET Core specific APIs go): https://github.com/dotnet/corefx/blob/c015db7bddfe9d7b54b795ff5861ccaa47c682a6/src/System.Memory/tests/System.Memory.Tests.csproj#L8

Reply:
Thanks, done